Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM ubuntu:xenial as app
 
 RUN apt update && \
-  apt install -y git-core language-pack-en python3.5 python3-pip python3-dev libmysqlclient-dev libffi-dev libssl-dev build-essential gettext openjdk-8-jdk && \
-  pip3 install --upgrade pip setuptools && \
+  apt-get install -y software-properties-common && \
+  apt-add-repository -y ppa:deadsnakes/ppa && apt-get update && \
+  apt install -y git-core language-pack-en python3.5 python3-pip python3-dev python3.8-dev python3.8-distutils libmysqlclient-dev libffi-dev libssl-dev build-essential gettext openjdk-8-jdk && \
```

Comment:
It looks like this file was added for future use in prod and decentralized devstack; can we skip installation of `python3.5` and `python3-dev`?  I don't think anything would still be using them after this update.